ui/lib/core/addon/components/selectable-card.hbs

```diff
@@ -0,0 +1,52 @@
+{{#if @hasSearchInput}}
+  <div class="selectable-card is-rounded no-flex" data-test-selectable-card-form={{@cardTitle}}>
+    <div class="is-flex-between is-fullwidth card-details">
+      <h3 class="title is-5" data-test-selectable-card>{{@cardTitle}}</h3>
+    </div>
+    <div class="has-top-bottom-margin">
+      <p class="sub-text">{{@subText}}</p>
+    </div>
+    <form class="is-flex" data-test-selectable-card={{@cardTitle}}>
+      <SearchSelect
+        class="is-flex-1"
+        @selectLimit="1"
+        @models={{@models}}
+        @backend={{@backend}}
+        @placeholder={{@placeholder}}
+        @disallowNewItems={{@disallowNewItems}}
+        @onChange={{this.handleInput}}
+        @fallbackComponent="input-search"
+        data-test-selectable-card-input={{@cardTitle}}
+      />
+      <button
+        type="button"
+        class="button is-secondary has-left-margin-s"
+        disabled={{unless this.value true}}
+        {{on "click" this.transitionToPage}}
```

Comment:
Best practice for buttons within a `form` is to have `type="submit"` (unless it's a cancel button) and then add `{{on 'submit' this.myAction}}` to the `<form>` element. That way out of the box the user can use the Enter key to submit the form

Reply:
I definitely agree!

When I did that I ran into this error, do you have any ideas on how to fix that? :)
<img width="1593" alt="Screen Shot 2023-01-13 at 1 09 41 PM" src="https://user-images.githubusercontent.com/30884335/212419598-95fc881f-9185-46bf-8bc5-d6fc9e049880.png">